variables.default

```diff
@@ -6,8 +6,8 @@ INPUT_PORT_INFLUX=4444
 INPUT_PORT_PROM=6666
 INPUT_PORT_ES=7777
 AOS_LOGIN=admin
-AOS_PASSWORD=admin
+AOS_PASSWORD=aos-aos
 AOS_PORT=443
 
 GRAFANA_LOGIN=admin
-GRAFANA_PASSWORD=admin
+GRAFANA_PASSWORD=aos-aos
```

Comment:
`variables.default` file doesn't seem to be used anywhere, instead according to `docker-compose.yaml` grafana password is set from the `./config/grafana.ini`. I guess grafana password should also be changed in `./config/grafana.ini` (https://github.com/Apstra/aosom-streaming/blob/master/config/grafana.ini#L131)

Reply:
https://github.com/Apstra/aosom-streaming/blob/master/README.rst#aosom-streaming-configuration

Password is changed in both files.

Reply:
variables.default should be copied to variables.env and modified accordingly. The makefile then uses these values even for grafana.